flutter/packages/duck_router/test/src/duck_router_test.dart

```diff
@@ -54,7 +54,7 @@ void main() {
       expect(locations2.uri.path, '/home');
     });
 
-    testWidgets('Resets to root', (tester) async {
+    testWidgets('Pops until', (tester) async {
```

Comment:
suggestion: would this be easier to follow with 'pops until X ?'

Reply:
```suggestion
    testWidgets('Pops until X', (tester) async {
```

Reply:
discussed offline, leave as is; :)